documentation/htc_workloads/managing_data/data-transfer.md

```diff
@@ -0,0 +1,167 @@
+
+...other submit file details...
+```
+
+
+
+## Submit File Example for many `/protected` Input/Output Files per Job
+
+If you have several output files being sent to `/protected`, you may wish to define a new submit file variable to avoid having to re-write the stash:/// path repeatedly. For example,
+
+```
+log = my_job.$(Cluster).$(Process).log
+error = my_job.$(Cluster).$(Process).err
+output = my_job.$(Cluster).$(Process).out
+
+requirements = (OSGVO_OS_STRING =?= "RHEL 7")
+
+STASH_LOCATION = stash:///ospool/protected/<username>
+transfer_input_files = $(STASH_LOCATION)/InputFile.txt
+transfer_output_remaps = "file1.txt = $(STASH_LOCATION)/file1.txt; file2.txt = $(STASH_LOCATION)/file2.txt; file3.txt = $(STASH_LOCATION)/file3.txt"
+
+...other submit file details...
+```
+
+# Phase Out of /public and Stashcp Command¶
+
+* As of Fall 2022, it is recommended that users use the new /protected file storage location instead of /public. 
+
+* Historically, output files could be transferred from a job to a `/public` location using the stashcp command within the job's executable, however, this mechanism is no longer encouraged for OSPool users. Instead, jobs should use `transfer_output_remaps` (an HTCondor feature) to transfer output files to `/public`. By using `transfer_output_remaps`, HTCondor will manage the output data transfer for your jobs. Data transferred via HTCondor is more likely to be transferred successfully and errors with transfer are more likely to be reported to the user.
+
+
+# Usage And Policies¶
+
+OSG staff reserve the right to monitor and/or remove data without notice to the user if doing so is necessary for ensuring proper use or to quickly fix a performance or security issue. Additionally, users should not use OSG resources or services for long-term data storage. Files and directories that have not been accessed for over six months may be deleted by OSG staff with or without notifying the user. See Policies for Using OSG via OSG Connect Submit Servers for more details.
+
+* `/home`: ALL JOBS MUST ALWAYS BE SUBMITTED FROM WITHIN /home. Users are also prohibited from making their `/home` directory world-readable due to security concerns. 
+* `/protected` : JOBS MUST NEVER BE SUBMITTED FROM WITHIN /protected. 
+
+
+# Questions and Get Help¶
+
+For additional data information, see the "Data Storage and Transfer" section of our FAQ.
+
+For assistance or questions, please email the OSG Research Facilitation team at support@osg-htc.org.
+
+
+
+-----
+
+
+User directories within /home are meant for general-use storage of your files needed for job submission. The initial quota per user is 50 GBs, and can be increased by request to support@osg-htc.org when a user needs more space for appropriately-sized files.
+
+If you're unable to submit jobs or your jobs are going on hold because you've reached your /home quota, please contact us at support@osg-htc.org about a quota increase.
```

Comment:
```suggestion
User directories within /home are meant for general-use storage of your files needed for job submission. The initial quota per user is 50 GBs, and can be increased by request to support@osg-htc.org when a user needs more space for appropriately-sized files.

If you're unable to submit jobs or your jobs are going on hold because you've reached your /home quota, please contact us at support@osg-htc.org about a quota increase.
```